src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,259 @@
+  - fugu
+  - behind-a-flag
+  - notification-triggers
+---
+
+{% Aside %}
+  The Notification Triggers API, part of Google's
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  is currently in development. This post will be updated as the
+  implementation progresses.
+{% endAside %}
+
+## What are Notification Triggers? {: #what }
+
+Web developers can display notifications using the
+[Web Notifications API](https://www.w3.org/TR/notifications/). This feature is often used
+with the [Push API](https://w3c.github.io/push-api/) to inform the user of
+time-sensitive information, such as breaking news events or received messages. Notifications are
+shown by running JavaScript on the user's device.
+
+The problem with the Push API is that it's not reliable for triggering notifications which _must_ be
+shown when a particular condition, like time or location, is met. An example of a _time-based
+condition_ is a calendar notification that reminds you of an important meeting with your boss at
+2:00 PM. An example of a _location-based condition_ is when you enter the vicinity of your grocery
+store, you get a reminder notification to buy milk. Network connectivity or battery-preserving
+features like doze mode can delay the delivery of push based notifications.
+
+Notification Triggers solve this problem by letting you schedule notifications with their triggering
+condition in advance, so that the operation system will deliver the notification at the right time
+even if there is no network connectivity or the device is in battery saver mode.
+
+{% Aside %}
+  For now, only _time-based triggers_ are supported in Chrome. Additional triggers, such as
+  location-based triggers, for example, will potentially be added in the future based on developer
+  demand.
+{% endAside %}
+
+### Use cases {: #use-cases }
+
+Calendar applications can use time-based notification triggers to remind a user of upcoming
+meetings. The default notification scheme for a calendar app could be to show a first "heads up"
+notification one hour before a meeting and then another more urgent notification five minutes
+before.
+
+A TV network might remind users that their favorite TV show is about to start, or a conference live
+stream is about to begin.
+
+Time zone conversion sites can use time-based notification triggers to let their users schedule
+alarms for telephone conferences or video calls.
+
+## Current status {: #status }
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| **4. Origin trial**                      | **[In Progress][ot]**    |
+| 5. Launch                                | Not started              |
```

Comment:
FYI this is strange to me. We are listing the steps sequentially, which implies that they need to happen in order. But step 2 hasn't been started, while step 3 and 4 are already in progress. cc @tomayac @jpmedley @petele

Reply:
@petele Did you modify the order in your article template on purpose? It differs from the [diagram](https://developers.google.com/web/updates/capabilities#process). I add a comment in the template, so we can follow up there.